### PR TITLE
Comparison between accuracies of SPORF and sklearn RFClassifier under default parameter and optimized hyperparameter conditions

### LINKED ISSUE #3: No comparison between SPORF and sklearn RFClassifier
**Description**
Currently, there is no official comparison using multiple datasets between the accuracies of SPORF and sklearn's RFClassifier. 

**Enhancement**
I will make Jupyter Notebooks and .py files that benchmark SPORF and RF using the CC-18 and 100 datasets in OpenML.

### PR DESCRIPTION
**Pull request by sju777**
**Reference Issues/PRs**

- Fixes #3 in terms of comparing both classifiers under default parameter and optimized hyperparameter conditions

**What does this implement/fix? Explain your changes.**

- Gather accuracy and runtime data in benchmarking for SPORF and sklearn RF classifier using the OpenML CC-18 and 100 suites
- Generate figures to compare accuracy and runtime data between SPORF and RF under default parameter and optimized hyperparameter conditions

**Any other comments?**

- Because a lot of the datasets took an incredibly long time to run (12+ hours), even on MARCC, I chose 12 smaller datasets to benchmark using optimized hyperparameters. That said, from the comparisons I was able to perform, it is fairly clear that SPORF performs more accurately than RF (SPORF performed better for 11 out of 12 datasets).

PR is complete, but any and all feedback is welcome!

### CODE REVIEW

**Comment by j1c** [comment]
Can you show me the marcc bash file? The clusters should never time out

**Comment by sju777** [comment]
> Can you show me the marcc bash file? The clusters should never time out

**Comment by sju777** [comment]
> Can you show me the marcc bash file? The clusters should never time out

I just uploaded my marcc bash file and the .py file I was running with it!

**Comment by j1c** [comment]
Idk how far along you are, but I would change the option to below. This will run at most 3 days.

```#SBATCH --time=3-0:0:0
#SBATCH --partition=lrgmem
#SBATCH --nodes=1
# number of tasks (processes) per node
#SBATCH --ntasks-per-node=48
#SBATCH --mem=500G

**Comment by sju777** [comment]
> Idk how far along you are, but I would change the option to below. This will run at most 3 days.
> 
> ```
> #SBATCH --partition=lrgmem
> #SBATCH --nodes=1
> # number of tasks (processes) per node
> #SBATCH --ntasks-per-node=48
> #SBATCH --mem=500G
> ```

I am currently trying to get the last 5 datasets of the OpenML CC18 set, but if it times out again, I will use the options you gave me.
